From 097e0db8ca890cca09b07178c68d13b486f71e0d Mon Sep 17 00:00:00 2001
From: Jami Kettunen <jami.kettunen@protonmail.com>
Date: Sun, 25 Aug 2024 14:50:25 +0300
Subject: [PATCH] Add missing GENERATE_PLUGIN_SOURCE to merkuro_contact_plugin

Launching merkuro-calendar was failing with:

  QQmlApplicationEngine failed to load component
  qrc:/main.qml:328:19: Type MainDrawer unavailable
  qrc:/MainDrawer.qml:341:9: Type CheckableCollectionNavigationView unavailable
  qrc:/CheckableCollectionNavigationView.qml:11:1: Failed to extract plugin meta data from '/usr/lib/qt6/qml/org/kde/merkuro/contact/libmerkuro_contact_pluginplugin.so': '/usr/lib/qt6/qml/org/kde/merkuro/contact/libmerkuro_contact_pluginplugin.so' is not a Qt plugin (metadata not found)
---
 src/contacts/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/contacts/CMakeLists.txt b/src/contacts/CMakeLists.txt
index b996e2ed..8f1d4494 100644
--- a/src/contacts/CMakeLists.txt
+++ b/src/contacts/CMakeLists.txt
@@ -76,6 +76,7 @@ target_sources(merkuro_contact_plugin PRIVATE
 
 ecm_add_qml_module(merkuro_contact_plugin
     URI "org.kde.merkuro.contact"
+    GENERATE_PLUGIN_SOURCE
     RESOURCES
         resources/fallbackBackground.png
 )
-- 
2.46.0

